README.md

```diff
@@ -28,6 +27,15 @@ ChugSplash makes it easy to deploy and upgrade smart contracts **securely**. It'
 * Deploys contracts at the same addresses across networks via `CREATE2`.
 * Generates deployment artifacts in the same format as hardhat-deploy.
 
+## Documentation
+
+1. [ChugSplash File](https://github.com/chugsplash/chugsplash/blob/develop/docs/chugsplash-file.md): Detailed explanation of the file where you define a deployment or upgrade. Start here.
+2. [Defining Variables in a ChugSplash File](https://github.com/chugsplash/chugsplash/blob/develop/docs/variables.md): Comprehensive reference that explains how to assign values to every variable type in a ChugSplash file.
+
+## Reach out
+
+If you have questions or want to request a feature, join our [Discord channel](https://discord.com/channels/1053048300565188729/1053048301143986219)!
```

Comment:
This needs to be an actual Discord invite link. You can get one by clicking the server name at the top left and then selected "Invite People".